pulumi/grapl/__main__.py

```diff
@@ -209,6 +209,12 @@ def main() -> None:
 
     observability_env_vars = observability_env_vars_for_local()
 
+    # This Google client ID is used by grapl-web-ui for authenticating users via Sign In With Google.
+    # TODO: This should be moved to Pulumi config somehwo, but I'm not sure the best way to do that atm.
+    google_client_id = (
+        "340240241744-6mu4h5i6h9j7ntp45p3aki81lqd4gc8t.apps.googleusercontent.com"
+    )
+
```

Comment:
I assume we don't want this leaking?

Reply:
client ID is fine, there's a secret we wouldn't want to expose, but it's not needed here.